source/gateway.go

```diff
@@ -44,6 +44,10 @@ import (
 const (
 	gatewayGroup = "gateway.networking.k8s.io"
 	gatewayKind  = "Gateway"
+	// GatewayAPIDualstackAnnotationKey is the annotation used for determining if a Gateway Route is dualstack
+	GatewayAPIDualstackAnnotationKey = "external-dns.alpha.kubernetes.io/dualstack"
```

Comment:
this does not need to be exposed as far as I see

Reply:
Good catch fixed (both)